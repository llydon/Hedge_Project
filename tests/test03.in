10 5 0.3
0.96 0.00 0.77 0.55 0.04 0.56 0.63 0.32 0.41 0.46
0.96 0.01 0.41 0.70 0.75 0.33 0.72 0.99 0.05 0.36
0.91 0.22 0.37 0.14 0.31 0.78 0.12 0.30 0.71 0.60
0.95 0.02 0.06 0.92 0.92 0.26 0.93 0.29 0.36 0.54
0.14 0.16 0.54 0.89 0.27 0.51 0.67 0.20 0.28 0.16
