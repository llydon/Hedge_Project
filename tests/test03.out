Expert probabilities: 
0.100000 0.100000 0.100000 0.100000 0.100000 0.100000 0.100000 0.100000 0.100000 0.100000
0.086017 0.114726 0.091062 0.097275 0.113357 0.096984 0.094968 0.104224 0.101448 0.099938
0.074972 0.132969 0.093608 0.091663 0.105227 0.102117 0.088954 0.090028 0.116177 0.104284
0.064917 0.141615 0.095308 0.099994 0.109083 0.091938 0.097623 0.093608 0.106816 0.099098
0.056327 0.162417 0.108004 0.087546 0.095504 0.098118 0.085214 0.099005 0.110627 0.097239
